package.json

```diff
@@ -7,7 +7,8 @@
     "clean": "rimraf dist",
     "start": "webpack-dev-server --mode=development",
     "test": "jest",
```

Comment:
Jest is complaining that it doesn't understand typescript. Something, probably involving babel, is not wired up correctly.